arnold/crud/read/step.py

```diff
@@ -66,11 +69,11 @@ def query_steps(
         .skip(skip)
         .limit(limit)
     )
-    return parse_obj_as(List[Step], list(raw_steps))
+    return [Step.model_validate(step) for step in raw_steps]
 
 
 def find_sample_fields(adapter: ArnoldAdapter) -> list[str]:
```

Comment:
Use get since it returns something.

Reply:
I will do the step refactoring in a separate PR - since it was a bit bigger than the others and more complex